libs/pubsub/query/query.go

```diff
@@ -95,6 +95,13 @@ func (q *Query) Matches(events map[string][]string) (bool, error) {
 	return q.matchesEvents(ExpandEvents(events)), nil
 }
 
+func (q *Query) MatchesEvents(events []types.Event) (bool, error) {
```

Comment:
@sergio-mena `MatchesEvents` is added to avoid bigger diff when apply origin [Matches](https://github.com/mmsqe/tendermint/blob/7a40bd5b891203aed369a3ffc11f66f15e6a9d1e/libs/pubsub/query/query.go#L91), let me know if need changes.

Reply:
This seems fine to me

Reply:
I also find this OK.
I have a question though: is adding `MatchesEvents` a temporary way to reduce diff size? Or will it stay when all PRs needed are backported to his branch?